### PR TITLE
getting started on testnet4

### PR DESCRIPTION
**Pull request by plebhash**
cc @GitGab19

### CODE REVIEW

File: src/getting-started.md [hunk 1]
```diff
@@ -79,15 +79,15 @@ Download a release from Sjors' fork of Bitcoin Core from https://github.com/Sjor
...
+testnet4=1
 server=1
 rpcuser=username
 rpcpassword=password
 ```
 Run the Template provider:
 
 ```bash
-./bitcoin-sv2-tp-0.1.2/bin/bitcoind -sv2 -sv2port=8442 -debug=sv2 
+./bitcoin-sv2-tp-0.1.2/bin/bitcoind -testnet4 -sv2 -sv2port=8442 -debug=sv2
```
Comment (GitGab19) [inline src/getting-started.md:90]: I guess that we should decide if setting testnet4 in config or by flag.
I would set it by using flag in the command to run bitcoind

Comment (plebhash) [reply src/getting-started.md:90]: if we don't change anything in the `bitcoin.conf` file, leaving the current `testnet=1` line, we get this:
```
$ ./src/bitcoind -testnet4 -datadir=/Users/plebhash/.bitcoin -sv2 -sv2port=8442
Error: Invalid combination of -regtest, -signet, -testnet, -testnet4 and -chain. Can use at most one.
```

I see your point around redundancy, so we have two alternatives to mitigate it:
- completely remove this line from `bitcoin.conf`
- replace it with `testnet4=1` (like the PR is already doing) but remove it from the CLI flag

Comment (GitGab19) [reply src/getting-started.md:90]: Yeah exactly, I would remove the line in bitcoin.conf, specifying the testnet4 as a flag

Comment (plebhash) [reply src/getting-started.md:89]: done

**Review by GitGab19 — APPROVED** [review]

**Pull request merged**